includes/actions/update-next-payment.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace AutomateWoo_Subscriptions;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Change a subscription's next payment date.
+ *
+ *
+ * @class Action_Subscription_Update_Next_Payment
+ * @since 1.2.2
+ */
+class Action_Subscription_Update_Next_Payment extends Abstract_Action_Subscription {
+
+	/**
+	 * The next payment date of the subscription before running the action.
+	 *
+	 * @var string
+	 */
+	private $original_payment_date;
+
+	/**
+	 * Explain to store admin what this action does via a unique title and description.
+	 */
+	public function load_admin_details() {
+		parent::load_admin_details();
+		$this->title       = __( 'Update Next Payment', 'automatewoo-subscriptions' );
```

Comment:
```suggestion
		$this->title       = __( 'Update Next Payment Date', 'automatewoo-subscriptions' );
```
I think 'Date' need to be included in the title

Reply:
Fixed in [`b1a0cd1`](https://github.com/woocommerce/automatewoo-subscriptions/pull/59/commits/b1a0cd1fdd453712bbbab10895e009e6096d05f1).